Kickstarter-iOS/Features/PaymentMethods/Controller/PaymentMethodsViewController.swift

```diff
@@ -143,6 +153,66 @@ internal final class PaymentMethodsViewController: UIViewController, MessageBann
 
   // MARK: - Private Helpers
 
+  private func goToPaymentSheet(data: PaymentSheetSetupData) {
+    PaymentSheet.FlowController
+      .create(
+        setupIntentClientSecret: data.clientSecret,
+        configuration: data.configuration
+      ) { [weak self] result in
+        guard let strongSelf = self else { return }
+
+        switch result {
+        case let .failure(error):
+          // strongSelf.viewModel.inputs.shouldCancelPaymentSheetAppearance(state: true)
+          strongSelf.messageBannerViewController?
```

Comment:
I think we can remove this comment
```suggestion
          strongSelf.messageBannerViewController?
```

Reply:
Oops yea I should made more formal comments like: